src/Function.h

```diff
@@ -312,6 +316,10 @@ class Function {
     /** If the Function has dimension requirements, check that the given argument
      * is compatible with them. If not, assert-fail. (If there are no dimension requirements, do nothing.) */
     void check_dims(int dims) const;
+
+    /** Define the output buffers. If the Function has types specified, this can be called at
+     * any time. If not, it can only be called for a Function with a pure definition. */
+    void define_output_buffers(const std::vector<Type> &types, int dims) const;
```

Comment:
"define" is a weird verb here. Would "create" be better?

Reply:
done